kapitan/refs/secrets/gpg.py

```diff
@@ -143,11 +153,24 @@ def _encrypt(self, data, fingerprints, encode_base64):
         else:
             raise GPGError(enc.status)
 
+    @classmethod
+    @lru_cache(maxsize=256)
+    def decrypt_with_cache(cls, data):
+        return gpg_obj().decrypt(data, **GPG_KWARGS)
+
     def _decrypt(self, data):
         """decrypt data"""
-        dec = gpg_obj().decrypt(data, **GPG_KWARGS)
+        dec = self.decrypt_with_cache(data)
```

Comment:
Maybe it's better to do the caching in the place that calls `_decrypt` already?

Reply:
You’re right. I was afraid caching takes place per instance method but it does so as a class. Will correct this